### DISCUSSION TITLE
v1.0.1

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/pogoplusle#18] [Announcements]
* Full bag notifications will now only alert once, unless bypassing Do Not Disturb is checked in system notification settings. (#14)
* Bluetooth pairing assistant now properly support more devices, including Samsung Galaxy S9 (Android 9, 10). Thanks to help from [u/nzAndy37](https://www.reddit.com/r/GoPlus/comments/x37yue/comment/j4569is/) from [247 Catcher](https://247catcher.com/) and Sauce Labs.
* Fix notifications being grouped on certain devices.
* Chinese translations support contributed by Jenny from [247 Catcher](https://247catcher.com/).
* Other minor fixes.

**Documentation update**: We now have a [device compatibility table for Bluetooth pairing assistant](https://github.com/Mygod/pogoplusle/wiki/Device-compatibility-table-for-Bluetooth-pairing-assistant). Feel free to contribute your data point(s) by commenting on the respective [release threads in discussions](https://github.com/Mygod/pogoplusle/discussions/categories/announcements)!

**Full Changelog**: https://github.com/Mygod/pogoplusle/compare/v1.0.0...v1.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/pogoplusle/releases/tag/v1.0.1'>v1.0.1</a>.</em>

**Comment by Marioysikax** [comment]
This appears to be the same version as 1.0.0-g on Play store. 
If so, all features seem to work perfectly on ROG phone 5 running Android 12. Root one not tested.

**Reply by Mygod** [reply]
This corresponds to 1.0.1-g on Google Play. Two versions are feature-wise identical, except that the Google version only has Google Play in-app purchase as the only donation option. Thanks for your report!
